operator/src/main/java/oracle/kubernetes/operator/helpers/ClientPool.java

```diff
@@ -53,7 +58,10 @@ public static ClientPool getInstance() {
 
   @Override
   protected ApiClient create() {
-    return getApiClient();
+    // disable pooling and always return the same instance
```

Comment:
i would like to see a comment here explaining why - for future generations of developers

Reply:
Can do